packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileInfoTiles.tsx

```diff
@@ -137,47 +157,109 @@ export const ProfileInfoTiles = () => {
   const {
     user_id,
     current_user_followee_follow_count,
-    allow_ai_attribution: hasAiAttribution
+    supporting_count,
+    allow_ai_attribution: hasAiAttribution,
+    supporter_count
   } = useSelectProfile([
     'supporting_count',
-    'user_id',
+    'supporter_count',
     'current_user_followee_follow_count',
+    'user_id',
     'allow_ai_attribution'
   ])
+  const { data: supporterRecords = [] } = useSupporters({
+    userId: user_id,
+    pageSize: MAX_PROFILE_TOP_SUPPORTERS
+  })
+
+  const supporters = useMemo(() => {
+    return supporterRecords.map((supporter) => supporter.sender)
+  }, [supporterRecords])
+
+  const { data: supportedUserRecords = [] } = useSupportedUsers({
```

Comment:
how do you feel about having component tiles for each datum that handle their own fetching and loading state? i like this approach too just wondering.

Reply:
Yeah good suggestion. I'm trying to strike a balance between when is enough code to warrant creating an entirely new component. There's a weird interaction here between the counts that exist on the user object (which determine whether we even try to render some of these tiles) and fetching the actual list of users for each.
I'm going to refine this a little bit

Reply:
[Updated it](https://github.com/AudiusProject/audius-protocol/pull/11421/commits/df956cbe7a5038c235ead514dc55ecbe26cf6bb9)